### ISSUE TITLE
Only send web notifications for emails _only_ to you

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#906]
Similar to "Focused inbox" (or combined with that), it would be nice to only receive notifications for really important emails, which are:

- Only to you (no one else in "to" or in "cc").
- Not from a noreply address

Is that possible at all?

**Comment by ChristophWurst** [comment]
> Only to you (no one else in "to" or in "cc").

:+1: 

> Not from a noreply address

:-1: you'd miss important emails like Github's "A personal access token has been added to your account" emails which are also sent from a `noreply` address.

**Comment by jancborchardt** [comment]
Ok, I’m fine with that. :) Small steps.

**Comment by arunikayadav42** [comment]
Can start working on this issue :) ?

**Comment by ChristophWurst** [comment]
> Can start working on this issue :) ?

Yes, of course. I'm looking forward to reviewing your pull request :rocket:

**Comment by arunikayadav42** [comment]
I am new to open source, so can you help please as to where to begin?

**Comment by ChristophWurst** [comment]
@arunikayadav42 awesome! I think reading through https://docs.nextcloud.com/server/13/developer_manual/general/index.html a bit is a good start to learn about Nextcloud. For the GitHub workflow, you could take a look at https://help.github.com/articles/about-pull-requests/ :wink:
